chart/kubeapps/README.md

```diff
@@ -0,0 +1,174 @@
+# <img src="https://github.com/kubeapps/kubeapps/raw/master/docs/img/logo.png" width="40" align="left"> Kubeapps
+
+[![Build Status](https://travis-ci.org/kubeapps/kubeapps.svg?branch=master)](https://travis-ci.org/kubeapps/kubeapps)
+
+[Kubeapps](https://kubeapps.com) is a web-based UI for deploying and managing
```

Comment:
why are you splitting the lines manually, is that a requirement for chart docs?

Reply:
No specific reason, I reverted to be more consistent